### ISSUE TITLE
normalize

### ISSUE THREAD

**Issue by regainOWO** [ElliotVincent/SitsSCD#2]
Thanks for your great work! i have some question to ask you
1. How did you get DynamicEarthNet dataset mean and std? I found is different from the regular ImageNet Dataset
``` python
self.mean = torch.tensor([83.1029, 80.7615, 69.3328, 133.8648], dtype=torch.float16).reshape(4, 1, 1)
self.std = torch.tensor([33.2714, 25.5288, 23.9868, 30.5591], dtype=torch.float16).reshape(4, 1, 1)
```
2. If i have a new satellite image, but is 16bit, What method is used for 16bit image to 8bit image that i could train my custom model?

**Comment by ElliotVincent** [comment]
Hi regainOWO,

1. The original DynamicEarthNet is indeed in 16bit. This is heavy for storage and loading. For this reason, I have compressed the dataset in 8bit (min-max stretch + gamma correction, jpeg compression). This version of the dataset is [available here for download](https://drive.google.com/file/d/1cMP57SPQWYKMy8X60iK217C28RFBkd2z/view?usp=drive_link). The mean and std used here are computed on the train set of my compressed version of DynamicEarthNet. Note that experimentally, the performance of methods are very similar whether using the original dataset or my lightweight version.

2. Now, you can also train my method (or any method) on a 16bit dataset. But training and inference should be done on same-type data. I recommend using the original data if storage/loading are not an issue for you.

**Comment by regainOWO** [comment]
Thank you!
